### DISCUSSION TITLE
[Poll] Would you join if we organize regular Q&A sessions ?

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#805] [General]
We plan to organize regular Q&A sessions to answer your questions and discuss together new stuff.

Please share your opinion with us to gauge the level of interest with 

- 👍 for "Yes, sure thing" 

 - 👎 for "Nope" 

below this message. 

As extra, you can also suggest platforms you might like to use for these sessions.

**Comment by WeberJulian** [comment]
I think having a Coqui-ai Youtube channel with tutorials and live Q&A would be interesting :)
A bit like [what Rasa is doing](https://www.youtube.com/c/RasaHQ/videos?view=0&sort=p&flow=grid).
Having videos on broad TTS/STT subjects (not coqui specific) would allow for new users to discover the company/repos.

**Reply by olafthiele** [reply]
It is still hard to search Youtube and it would be great if people also found those answers by searching Google. Unfortunately, I am not aware of a good plattform to do that. But anything is better than nothing :-)

**Reply by WeberJulian** [reply]
hum, yeah maybe it would be interesting to have searchable transcription/summaries of these Q/As hosted on the coqui-ai? Like a blog post?

**Reply by olafthiele** [reply]
Just an idea ...

**Reply by thorstenMueller** [reply]
But in general i'd be happily a part of Youtube livestream or recorded sessions about 🐸 TTS.

**Reply by yuananf** [reply]
Is it possible to make a discord channel? I joined Uberduck and vocodes channel, and the discussion is really helpful.

**Reply by erogol** [reply]
There is a Gitter channel along the same lines. Are there any pros to using discord over gitter ?

**Reply by WeberJulian** [reply]
I think the main thing that discord has over Gitter is voice channels, the ability to present with screen sharing, and to allow for questions (text/voice).

**Reply by erogol** [reply]
Element has all these too but we need to wait them to merge Gitter :(

**Reply by thorstenMueller** [reply]
I'm not a big fan of switching communication channels (_again_) or adding another. Coqui previously switched communication channels and in my opinion we could "lose" some people who did not notice that there was another communication channel.

**Reply by WeberJulian** [reply]
Yeah I wouldn't recommend switching either, but having a complementary place to share things like YouTube is ok

**Reply by erogol** [reply]
There is also a timezone issue. I think it is better to organize asynchronously and collect the discussion points and questions in advance. Then release the video or transcript of the meeting so that everyone can make use of it from anywhere.

**Comment by erogol** [comment]
I think it makes sense to go with Zoom or Google meet and collect questions on Github Discussion. We can prioritize based on the number of 👍s. And we can release the meeting recording afterward. So questions are google indexable and answers would be in the video.

**Comment by erogol** [comment]
Just a follow-up on that....

Here are the specs for our 🐸 community meeting v0.5 ✨

I will answer the questions posted to the link below ordering by the votes. Feel free to post your questions starting from today.

and post the video recording somewhere if you miss the call

So let's see how it's gonna turn out 😄

-> Meeting link: https://meet.google.com/huz-jvii-ngs
-> Meeting time: Nov 25, 2021 Thursday 17:30 - 18:15 CET
-> Ask or upvote you questions here: https://github.com/coqui-ai/TTS/discussions/categories/qs-for-live-q-a
